cpp/tests/utilities/column_utilities.cu

```diff
@@ -69,5 +144,56 @@ void expect_equal_buffers(void const* lhs, void const* rhs,
                             typed_rhs));
 }
 
+struct column_view_printer {
+  template <typename Element, typename std::enable_if_t<is_numeric<Element>()>* = nullptr>
+  void operator()(cudf::column_view const& col, std::vector<std::string> & out) {
+    auto h_data = cudf::test::to_host<Element>(col);
+
+    out.resize(col.size());
+
+    if (col.nullable()) {
+      size_type index = 0;
+      std::transform(h_data.first.begin(), h_data.first.end(), out.begin(), [&h_data, &index](Element el) {
+          return (bit_is_set(h_data.second.data(), index++)) ? std::to_string(el) : std::string("@");
+        });
+    } else {
+      std::transform(h_data.first.begin(), h_data.first.end(), out.begin(), [](Element el) {
+          return std::to_string(el);
+        });
+    }
+  }
+
+  template <typename Element, typename std::enable_if_t<not is_numeric<Element>()>* = nullptr>
+  void operator()(cudf::column_view const& col, std::vector<std::string> & out) {
+    CUDF_FAIL("printing not currently enabled for non-numeric arguments");
```

Comment:
You should be able to use `to_host` for strings columns here: https://github.com/rapidsai/cudf/blob/branch-0.11/cpp/tests/utilities/column_utilities.hpp#L101

Reply:
I'll look at string support, it wasn't available when I started.